### ISSUE TITLE
Select menu - "faked" current selection text separately announced by screen readers

### ISSUE THREAD

**Issue by patrickhlauke** [jquery/jquery-mobile#8256]
Taking the first example from http://demos.jquerymobile.com/1.4.5/selectmenu/, based on the following simplified markup

```
<div ... class="...">
  <span>The 1st Option</span>
  <select name="..." ...>
    <option value="1">The 1st Option</option>
    ...
  </select>
</div>
```

when using TalkBack/Android or VoiceOver/iOS, the `<span>` is read/announced separately, before the actual `<select>`. This results in redundant/inappropriate output (as the `<span>` is not announced as being actionable, and the current option is already announced once the user moves their screenreader focus to the actual `<select>`).

Solution would be to apply an `aria-hidden="true"` to that `<span>`

```
<div ... class="...">
  <span aria-hidden="true">The 1st Option</span>
  <select name="..." ...>
    <option value="1">The 1st Option</option>
    ...
  </select>
</div>
```

which suppresses that element completely from being seen/announced by SRs

**Comment by arschmitz** [comment]
@patrickhlauke thank you for reporting this. PR very welcome if you have time. Either way we will make sure this is fixed for the beta coming shortly.

**Comment by patrickhlauke** [comment]
No worries, I'll see if I can find some time to properly PR this in the next few days :)
